slides/ds101-worskhop.mdx

```diff
@@ -15,9 +15,13 @@ export const theme = { ...CDTheme };
 
 ---
 
-# Backing a toast
-
-## Ice breaker
+<Center>
```

Comment:
It's using `<Center>` by default afaik so you can remove it

Reply:
Removed